mkdocs.yml

```diff
@@ -59,13 +58,20 @@ nav:
       - Connect to Wi-Fi: setup/provision.md
       - Multi-Robot Setup: setup/multi-robot.md
       - ROS 2 Network Config: setup/xml-config.md
-      - Setup Compute Boards:
+      - Connected Computers:
```

Comment:
Maybe a better headline is "ROS 2 Installation?" Interested in others' feedback.

Reply:
I feel like this might get confusing with other areas associated with ROS2 potentially. Maybe "Development", "Desktop", some other descriptor that describes how this is fundamentally different than an attached SOM/Compute.